robot_smach_states/examples/simple_navigate.py

```diff
@@ -13,15 +12,8 @@
     rospy.init_node('simple_navigate')
 
     robot_name = sys.argv[1]
```

Comment:
Add argument check like other files.

Reply:
See a71172c306547eba6d50c057baaef8c0ec10a1d0 and 106535870505f785620744139c45820da4f330a4